Hardware/CPU/IntelCPU.cs

```diff
@@ -339,6 +343,60 @@ public IntelCPU(int processorIndex, CPUID[][] cpuid, ISettings settings)
         }
       }
 
+      switch (microarchitecture) {
+        case Microarchitecture.SandyBridge:
+        case Microarchitecture.IvyBridge:
+        case Microarchitecture.Haswell:
+        case Microarchitecture.Broadwell:
+        case Microarchitecture.Skylake:
+        case Microarchitecture.KabyLake:
+          cStatesMSRs = new[] { (2, 0x60Du), (3, 0x3F8u), (6, 0x3F9u), (7, 0x3FAu) };

```

Comment:
No need for each of these numbers here, but it would be nice to have at least a reference to where this is coming from.